src/gui/settings/foundObjects.html

```diff
@@ -78,6 +146,9 @@ <h1 class="title">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&n
 <script>
     var previousFoundObjects = null; // stores stringified foundObjects so that we can detect changes over time
```

Comment:
Comment is out of date (or line 180 is wrong)

Reply:
Good catch! Fixing